docs/config/apache2-site

```diff
@@ -16,7 +16,7 @@
 
 	php_value include_path .:/usr/share/php:%LORISROOT%/project/libraries:%LORISROOT%/php/libraries
 
-	DirectoryIndex main.php index.html
+	#DirectoryIndex main.php index.html
```

Comment:
is the default index.php?

Reply:
The default is "index.html", but the last rewrite rule of "RewriteRule ^(.*)$" catches everything.